deps/chakrashim/lib/chakra_shim.js

```diff
@@ -671,6 +672,76 @@
 
       return ownPropertyNames;
     };
+
+    utils.beforeContext = function (contextGlobal) {
+      return Object_getOwnPropertyDescriptors(contextGlobal);
+    }
+
+    function descriptorDiff(a, b) {
+      if (!a || !b) {
+        return false;
+      }
+      if (a.value !== b.value && a.value === a.value && b.value === b.value) {
+        // allow for NaN in value
+        return false;
+      }
+      if (a.enumerable !== b.enumerable ||
+          a.configurable !== b.configurable ||
+          a.writable !== b.writable) {
+        return false
+      }
+      if (a.get !== b.get || a.set !== b.set) {
+        return false;
+      }
+      return true;
+    }
+    utils.afterContext = function (beforeDescriptors, contextGlobal, sandbox) {
+      try{
```

Comment:
nit: spacing around braces.  I think the linter on Windows should have caught it.

Reply:
Fixed.